### PR TITLE
Feature/view refactory

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: sale_commission/partner_view.xml [hunk 1]
```diff
@@ -8,9 +8,9 @@
             <field name="inherit_id" ref="base.view_partner_form"/>
             <field name="priority" eval="18"/>
             <field name="arch" type="xml">
-                    <field name="supplier" position="after">
-                        <field name="agent" string="Consultancy"/>
-                    </field>
+                   <xpath expr="//page/group/group/field[@name='supplier']" position="after">
```
Comment (pedrobaeza) [inline sale_commission/partner_view.xml]: Better to put:

```
<xpath expr="//page[@name='sales_purchases']//field[@name='supplier']" position="after">
```

It avoids ambiguity

File: sale_commission/partner_view.xml [hunk 2]
```diff
@@ -20,18 +20,15 @@
             <field name="inherit_id" ref="view_partner_form_add_agent"/>
             <field name="arch" type="xml">
                 <field name="user_id" position="after">
-                    <group colspan="4">
```
Comment (pedrobaeza) [inline sale_commission/partner_view.xml:23]: Keep group, but with string `Agents/Consultancy`

File: sale_commission/sale_agent_view.xml [hunk 3]
```diff
@@ -27,17 +27,23 @@
...
-                        <field attrs="{'required':[('type','=','comercial')], 'invisible':[('type','!=','comercial')]}" name="employee_id" select="2"/>
-                    </group>
-                    <field domain="[('parent_id','=',False),('type_tax_use','&lt;&gt;','purchase')]" name="retention_id"/>
-                    <!--label colspan="2"/-->
-                    <field name="active"/>
+                    <group>
+                    	<group>
+                    		<group>
+                    			
+                    			<field name="name" select="1"/>
```
Comment (pedrobaeza) [inline sale_commission/sale_agent_view.xml]: select is not used anymore

File: sale_commission/sale_agent_view.xml [hunk 4]
```diff
@@ -45,6 +51,8 @@
                             </group>
                             <group>
                                 <field name="commission"/>
+                                <field domain="[('parent_id','=',False),('type_tax_use','&lt;&gt;','purchase')]" name="retention_id"/>
+                    	<!--label colspan="2"/-->
```
Comment (pedrobaeza) [inline sale_commission/sale_agent_view.xml]: Remove this

File: sale_commission/partner_view.xml [hunk 5]
```diff
@@ -8,9 +8,9 @@
             <field name="inherit_id" ref="base.view_partner_form"/>
             <field name="priority" eval="18"/>
             <field name="arch" type="xml">
-                    <field name="supplier" position="after">
-                        <field name="agent" string="Consultancy"/>
-                    </field>
+                   <xpath expr="//page[@string='Sales &amp; Purchases']//field[@name='supplier']" position="after">
+                           <field name="agent" string="Consultancy"/>
+				   </xpath>
```
Comment (yvaucher) [inline sale_commission/partner_view.xml]: You have a mix of spaces and tab here.

Please use only spaces.

**Comment by pedrobaeza** [comment]
Now seems right :+1:

**Comment by rvalyi** [comment]
:+1:  LGTM

**Comment by omar7r** [comment]
:+1:

**Comment by mileo** [comment]
=]

Luis Felipe Miléo
KMEE
+55 35 8876-3663
+55 21 4063-8872
mileo@kmee.com.br
kmee.com.br
Em 02/09/2014 07:03, "omar7r" notifications@github.com escreveu:

**Comment by gurneyalex** [comment]
@mileo @omar7r why merge this on master? Is this odoo v8 specific?

**Comment by gurneyalex** [comment]
It does not seem to be. I'll merge back to 7.0 before removing master and creating the 8.0 branch.

**Comment by mileo** [comment]
Its 7.0

Luis Felipe Miléo
KMEE
+55 35 8876-3663
+55 21 4063-8872
mileo@kmee.com.br
kmee.com.br
Em 02/09/2014 16:16, "Alexandre Fayolle" notifications@github.com
escreveu:

**Comment by rvalyi** [comment]
@gurneyalex sorry I didn't pay attention it was targeting the wrong branch, you are right your proposal I think.

**Pull request merged**